changelogs/fragments/64958-routeros-facts-ipv6.yml

```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - routeros_facts - Prevent crash of module when ``ipv6`` package is not installed
+  - routeros_facts - Unit tests for the case when ``ipv6`` package is not installed
```

Comment:
You don't need a changelog fragment for the unit tests.

Reply:
Oh, OK. Removed it, thanks!